install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+if [[ $(whoami) != "root" ]]; then
+  echo "This file needs to be executed as root or with sudo."
```

Comment:
This "script" sounds better instead of "file."

Reply:
This script may not require root if the directory the user chooses is owned by them (e.g. `~/bin/`); could this check be moved down along with a permissions check to see if it is needed?